psse/psse-model/src/main/java/com/powsybl/psse/model/io/LegacyTextReader.java

```diff
@@ -123,14 +127,38 @@ public String readRecordLine() throws IOException {
         return newLine.toString().trim();
     }
 
-    // all the lines beginning with "@!" are record lines defining the attribute fields
-    private static boolean isRecordLineDefiningTheAttributeFields(String line) {
-        return line.length() >= 2 && line.substring(0, 2).equals("@!");
+    private static String removeComment(String line) {
+        return line.replaceAll(FileFormat.REMOVE_COMMENT_REGEX, "$1$2");
     }
 
-    private static String removeComment(String line) {
-        // Only outside quotes
-        return line.replaceAll("('[^']*')|(^/[^/]*)|(/[^/]*)", "$1$2");
+    // Compact spaces, remove spaces before the comma, and replace space with comma outside quoted text
+    private static String processText(String line) {
```

Comment:
@pjanecek0 @marqueslanauja is there a performance impact on a large file parsing by adding this ?

Reply:
No, there is no performance impact. I have imported a PSSE version 33 file with 23,500 buses 10 times, and no performance impact is observed. I have used `System.currentTimeMillis()` to track the time.

Time in milliseconds required for importing the case using the main version: 
`5412, 3133, 2536, 2379, 2506, 2393, 2324, 2353, 2317, 2330`, average `2768,3`

Time in milliseconds required for importing the case using the PR 3440 version: 
`5455, 3261, 2496, 2407, 2388, 2389, 2444, 2429, 2669, 2503`, average `2844,1`